Source/WebKit/UIProcess/WebAuthentication/WebAuthenticatorCoordinatorProxy.cpp

```diff
@@ -116,8 +116,8 @@ void WebAuthenticatorCoordinatorProxy::handleRequest(WebAuthenticationRequestDat
             RELEASE_LOG_ERROR(WebAuthn, "Request cancelled due to rejected prompt after lack of user gesture.");
         }
     };
-    
-    if (!data.processingUserGesture && data.mediation != MediationRequirement::Conditional && !m_webPageProxy.websiteDataStore().authenticatorManager().isVirtual())
+
+    if (data.mediation != MediationRequirement::Conditional && !m_webPageProxy.websiteDataStore().authenticatorManager().isVirtual())
         m_webPageProxy.uiClient().requestWebAuthenticationNoGesture(origin, WTFMove(afterConsent));
```

Comment:
You can remove anywhere in the code with `requestWebAuthenticationNoGesture` as this is the only place it's used.

Reply:
also the `afterConsent` block, the contents may just be moved.

Reply:
Also I think AuthenticatorCoordinator::resetUserGestureRequirement should be removed.